presto-docs/src/main/sphinx/connector/hana.rst

```diff
@@ -0,0 +1,100 @@
+====================
+HANA Connector
+====================
+
+The HANA connector allows querying and creating tables in an
+external HANA database. This can be used to join data between
+different systems like HANA and Hive, or between two different
+HANA instances.
+
+Configuration
+-------------
+
+To configure the HANA connector, create a catalog properties file
+in ``etc/catalog`` named, for example, ``hana.properties``, to
+mount the HANA connector as the ``hana`` catalog.
+Create the file with the following contents, replacing the
+connection properties as appropriate for your setup:
+
+.. code-block:: none
+
+    connector.name=hana
+    connection-url=jdbc:sap://[serverName[\instanceName][:portNumber]]
+    connection-user=root
+    connection-password=secret
+
+Multiple HANA Databases or Servers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The HANA connector can only access a single database within
+a HANA server. Thus, if you have multiple HANA databases,
+or want to connect to multiple instances of the HANA, you must configure
+multiple catalogs, one for each instance.
+
+To add another catalog, simply add another properties file to ``etc/catalog``
```

Comment:
```suggestion
To add another catalog, add another properties file to ``etc/catalog``
```
In documentation, avoid words like simple or easy. They almost never make the explanation better and are a distraction to the reader.